README.md

```diff
@@ -25,6 +25,20 @@ yarn test
 
 Once your changes have been merged to `dev`, it will immediately deployed to app.binary.com/beta.
 
+## Change Endpoint
```

Comment:
@brucebinary does QA really need this? We're developing a Chrome Extension for QA to change the endpoint.

Reply:
@ashkanx QA requested for an alternative cause they can't use the chrome extension in mobile devices.